### ISSUE TITLE
Panic goroutine stack exceeds 1000000000-byte limit

### ISSUE THREAD

**Issue by Vad1mo** [dmlittle/scenery#26]
Some of my TF plans fail with this stacktrace.

I used commit 6dcc97e938841a47240f140356e9fd38a7a37945

```
~ aws_dynamodb_table.dynamodb-table
    ttl.#:                         "0" => "1" 
runtime: goroutine stack exceeds 1000000000-byte limit
fatal error: stack overflow

runtime stack:
runtime.throw(0x60a8fc, 0xe)
	/usr/local/go/src/runtime/panic.go:608 +0x72
runtime.newstack()
	/usr/local/go/src/runtime/stack.go:1008 +0x729
runtime.morestack()
	/usr/local/go/src/runtime/asm_amd64.s:429 +0x8f

goroutine 1 [running]:
runtime.heapBitsSetType(0xc00044eaa0, 0x20, 0x18, 0x5de940)
	/usr/local/go/src/runtime/mbitmap.go:911 +0xa30 fp=0xc0201a6390 sp=0xc0201a6388 pc=0x415890
runtime.mallocgc(0x20, 0x5de940, 0xc00004ea01, 0xc0201a64e8)
	/usr/local/go/src/runtime/malloc.go:933 +0x540 fp=0xc0201a6430 sp=0xc0201a6390 pc=0x40c300
runtime.newobject(0x5de940, 0x20)
	/usr/local/go/src/runtime/malloc.go:1032 +0x38 fp=0xc0201a6460 sp=0xc0201a6430 pc=0x40c938
encoding/json.(*scanner).eof(0xc00044d540, 0x5efde0)
	/usr/local/go/src/encoding/json/scanner.go:137 +0x91 fp=0xc0201a6490 sp=0xc0201a6460 pc=0x527ad1
encoding/json.checkValid(0xc0201a6568, 0x0, 0x20, 0xc00044d540, 0x0, 0xc0201a6568)
	/usr/local/go/src/encoding/json/scanner.go:33 +0xd8 fp=0xc0201a64c0 sp=0xc0201a6490 pc=0x5279a8
encoding/json.Valid(0xc0201a6568, 0x0, 0x20, 0xc0201a6568)
	/usr/local/go/src/encoding/json/scanner.go:20 +0x59 fp=0xc0201a6500 sp=0xc0201a64c0 pc=0x5278a9
github.com/dmlittle/scenery/pkg/printer.formatValue(0x0, 0x0, 0x1e, 0x0, 0x0)
	/go/src/github.com/dmlittle/scenery/pkg/printer/printer.go:244 +0x79 fp=0xc0201a65c8 sp=0xc0201a6500 pc=0x534c99
...
...
```

**Comment by dmlittle** [comment]
@Vad1mo thanks for opening up the issue! Would you be able to provide a redacted output of that `aws_dynamodb_table.dynamodb-table` block that is currently causing the error? I _think_ I know what's causing it and I'll put a fix later today but I'd like to confirm I fixed the issue you're seeing.

**Comment by dmlittle** [comment]
@Vad1mo I believe this is fixed in master now (be3f5758831413a15bafa8902ff161d25af96d52). Can you confirm that is the case. If it's not, I'll reopen this issue.

**Comment by Vad1mo** [comment]
thx, for the quick fix. it works now!
